src/transport/SecureSession.cpp

```diff
@@ -28,13 +28,21 @@ Access::SubjectDescriptor SecureSession::GetSubjectDescriptor() const
         subjectDescriptor.authMode    = Access::AuthMode::kCase;
         subjectDescriptor.subject     = mPeerNodeId;
         subjectDescriptor.cats        = mPeerCATs;
-        subjectDescriptor.fabricIndex = mFabric;
+        VerifyOrDie(mEffectiveFabric == kUndefinedFabricIndex);
```

Comment:
This seems wrong. All CASE subjects should have a subject. It should be:

`VerifyOrDie(mEffectiveFabric != kUndefinedFabricIndex);`